docs/products/cassandra/concepts/tombstones.rst

```diff
@@ -0,0 +1,61 @@
+Tombstones in Apache Cassandra®
+===============================
+
+Apache Cassandra manages deletion of data via a mechanism called *tombstones*. Because Cassandra is a distributed system,
+it cannot delete data immediately in the same way as a traditional relational database. On a high-level, when a row is
+deleted, instead of immediately deleting it, Cassandra will mark it as a tombstone row. Then, as part of regularly scheduled
+maintenance, the row will actually get deleted. This maintenance is called *compaction* and the threshold is controlled by
+the table-level setting ``gc_grace_seconds``. Any tombstone older than this setting will be removed completely during
+compaction (with some caveats - more details in the Cassandra `documentation <compaction-tombstones_>`_ on compaction). By
+default, this value is 864000 seconds (10 days), which means that a single node can be unavailable for that long before you
```

Comment:
Can you expand a little on what resurrected data means. And what potential problems there are. It feels like a good reason to not lower the threshold too low.

Reply:
I've tried to make this a bit clearer, but could also just remove it.

Reply:
I think we are getting a little too deep in the cassandra internals for what is the purpose of our article. It is true that resurrected data is a big problem and in some cases it can go undetected for long time. It is worth a mention and a link to https://thelastpickle.com/blog/2016/07/27/about-deletes-and-tombstones.html.

Reply:
In the end I just removed this section since it was hard to make a statement that was both succinct and correct, and it's not central to the purpose of the article.